src/pages/payment/page.tsx

```diff
@@ -83,14 +85,14 @@ export default () => {
     return (
         <>
             <Vexile gap={3}>
-                <Header1 purple>{price}원</Header1>
+                <Header1 purple>{price + shippingFee}원</Header1>
```

Comment:
Variable `price` is calculated as wholePrice + shippingFee. You don't need to add shippinFee any more